resources/css/filament/admin/theme.css

```
@@ -1,4 +1,1042 @@
 @source '../../../../resources/views/filament/**/*';
+
+/* Filament's --font-family is set by ->font('Geist', provider: GoogleFontProvider::class)
+ * in each panel provider, which writes the value into an inline <style> tag via
+ * vendor/filament/filament/resources/views/components/layout/base.blade.php.
+ * That declaration wins over CSS :root rules, so we configure it through Filament's
+ * API rather than overriding it here. */
+
+/* Tailwind v4 theme tokens — declared once, both panels use them */
+@theme {
+  --font-sans:
+    "Geist", ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.11.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)